packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -478,8 +487,11 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     if (this._isExpanded) {
       this._dismissMenu();
     }
-    if (this.props.onClick) {
+
+    if (!this._processingTouch && this.props.onClick) {
```

Comment:
Now if we are not `processingTouch` but there also isn't a `this.porps.onClick` we will call `this._onMenuClick` where as before we would not call `this.onMenuClick`, is that expected?

Reply:
No you're correct, I've changed it to an else if to only do a menu click if we're processingTouch